### ISSUE TITLE
Adobe reader automatically tries to install

### ISSUE THREAD

**Issue by VarunAgw** [wavebox/waveboxapp#1144]
Wavebox Version: 10.0.25.2 stable
Operating System & Version: Windows 10 1909 x64

I guess it's chrome legacy. Whenever you create a new profile in chrome, adobe reader automatically install it's extension. This behaviour also got copied here.

Minor issue though.

**Comment by Thomas101** [comment]
It's because Wavebox advertises itself as a chromium-compliant extension browser. Adobe reader see's this and tries to add itself in. You should be able to remove it out and I don't think it will auto-add again :-/

**Comment by VarunAgw** [comment]
Yes. I did that. Was just a minor issue. Would be great if you disable this behaviour in first place if you get a chance.

**Comment by nmat** [comment]
in the latest wavebox 10 release I do not seem to be getting this. Is this still happening to you?
